lib/actions/BackendAction.js

```diff
@@ -45,13 +45,19 @@ class BackendAction {
     const pid = await utils.previousProcess('backend', this.settingsFolder)
     if (pid) throw new Error(`Backend process is already running with pid: ${pid}. Please quit this process first.`)
 
-    this.dcClient = new DcHttpClient(this.userSettings, logger)
+    this.dcClient = this.dcClient
+      ? this.dcClient
```

Comment:
Why do you check if the dcClient is already exisit?
The run function is the first function that get called when an action from the terminal is dispatched

Reply:
moved init to constructor, needs to be changeable for unit test